backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/services/LinksDescriptionRegistry.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.spring.collaborative.forms.services;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.eclipse.sirius.web.api.configuration.ILinksDescriptionRegistry;
+import org.eclipse.sirius.web.forms.description.FormDescription;
+
+/**
+ * Registry containing all the links descriptions. It actually contains form descriptions, these forms should contain
+ * only Links, possibly grouped within groups if needed.
+ *
+ * @author ldelaigue
+ */
+public class LinksDescriptionRegistry implements ILinksDescriptionRegistry {
```

Comment:
This can be removed too, it's only necessary for the special use case of the properties. We went with another approach in the ADR.